shared/attributes.asciidoc

```diff
@@ -181,6 +181,13 @@ Kibana app and UI names
 :a-data-source:      a data view
 
 
+//////////
+Enterprise Search
+//////////
+
+:crawler: Enterprise Search Crawler
```

Comment:
```suggestion
:crawler: Enterprise Search web crawler
```

Reply:
We call it "crawler" internally, but the official feature name is "web crawler". And lower case because feature names are lower case.

Reply:
Looks like I missed that in https://github.com/elastic/enterprise-search-pubs/pull/2168 :grin:

Reply:
😆